cfpb/cfgov-refresh
# Apply gulp-changed to style and script tasks

Adds [gulp-changed](https://github.com/sindresorhus/gulp-changed) to style and script tasks to prevent them from being run if the files at the destination are older than the source files.

Discussion welcome!

## Testing

1. Pull branch
2. `npm install`
3. Run `gulp styles` and note how long each subtask takes. (Don't rely on the final overall `styles` time, as it doesn't seem to include the time taken on each subtask.)
4. Run `gulp styles` again and note how it takes almost no time, since none of the styles need to be rebuilt!
5. Run `gulp scripts` a couple times and watch the time go way down.
6. Test various pages (especially things that use some of the one-off style and script tasks) to ensure they are still working

## Todos

- [x] Investigate testing the destination file within each different Gulp sub-task.
- [x] Add gulp-changed to script tasks (and others, if appropriate)

## Checklist
